blockchain/chain/src/store/chain_store.rs

```diff
@@ -66,12 +65,13 @@ pub struct ChainStore<DB> {
     /// Publisher for head change events
     publisher: Publisher<HeadChange>,
 
-    /// Tracks head change subscription channels
-    subscriptions: Arc<LockfreeMap<i64, Option<Receiver<HeadChange>>>>,
-
-    /// Keeps track of how many subscriptions there are in order to auto-increment the subscription ID
-    subscriptions_count: AtomicCell<usize>,
+    // XXX: 'subscriptions' disabled since it is unused.
```

Comment:
Can we either delete those or create an issue to use them again?

Reply:
I'll open a tracking issue.

Reply:
Tracking issue opened and added to source code: https://github.com/ChainSafe/forest/issues/1976